### PR TITLE
Fix lighthouse SEO issues 

### LINKED ISSUE #7: SEO issues from Lighthouse Audit
Screenshot 2020-09-10 at 16 26 23

- [ ] Document does not have a meta description
- [ ] Links do not have descriptive text
- [ ] Tap targets are not sized appropriately

### PR DESCRIPTION
**Pull request by bethcollins92**
## Done

- Add meta description
- Fix kubeflow-lite buttons according to [copy doc](https://docs.google.com/document/d/1Wvhl0yYV_w0BJsyZbQFKg_QV6RjMLbmQU3x0rsD2F4Q/edit#heading=h.mvjb0cu9g00d)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8046
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Run lighthouse audit check SEO is 100
- Check kubeflow-lite buttons match copy doc 

## Issue / Card

Fixes #7 

## Screenshots

image

### CODE REVIEW

**Comment by bethcollins92** [comment]
@anthonydillon in response to this [comment](https://github.com/canonical-web-and-design/charmed-kubeflow.io/pull/16#pullrequestreview-492502091), I copied the code you suggested, but should the "Install Kubeflow lite" be "Install kubeflow-lite" with a dash?

File: templates/base_layout.html [hunk 1]
```diff
@@ -40,9 +40,11 @@
   <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/f8097dea-Ubuntu-LI_W.woff2" crossorigin>
   <link rel="preload" as="font" type="font/woff2" href="https://assets.ubuntu.com/v1/fff37993-Ubuntu-R_W.woff2" crossorigin>
 
-  <meta name="description" content="{% block meta_description %}{% endblock %}">
+  
+  <meta name="description" content="{% block meta_description %}Open source Python operators for Kubeflow on-rails. For workstations, multi-cloud and edge.{% endblock %}">
```
Comment (anthonydillon) [inline templates/base_layout.html:44]: Can you add this description to the index page please.

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Thanks @bethcollins92 one small comment

File: templates/base_layout.html (hunk 1 above)
Comment (bethcollins92) [reply templates/base_layout.html:44]: Added it 👍

**Review by anthonydillon — APPROVED** [review]
Awesome thanks :+1:

**Pull request merged**